src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1154,8 +968,9 @@ public static function debug_data() {
 			);
 		}
 
-		$info['wp-constants']  = self::get_wp_constants();
+		$info['wp-server']     = self::get_wp_server();
 		$info['wp-database']   = self::get_wp_database();
+		$info['wp-constants']  = self::get_wp_constants();
 		$info['wp-filesystem'] = self::get_wp_filesystem();
```

Comment:
I reordered this, to be able to get rid of the `$info['wp-database']` in lines 199-202

Actually I would like to order the function calls bellow as well. OCD is itching.

This is the correct order of the function calls, and by having them correct, we can completely replace the original `$info` definition which was the plan of this.
And I'd like it best if we could still swap around the function defintions of `get_wp_database()` in line 1391 and `get_wp_constants()` in line 1238 to match this order.

Your thought @dmsnell ?

Reply:
After discussing in Slack, this all sounds great, and I apologize for reordering these before. The insertion order of the keys in the array determines the order they are displayed in `wp-admin`, and so we want to preserve the existing order.

Reply:
@apermo thoughts on this? when I dug into it, I felt like something ad-hoc was missing a bigger point, so I proposed encoding the ordering as a specific semantic instead of relying on how the code is already written.

https://github.com/WordPress/wordpress-develop/pull/7289

Reply:
@dmsnell updated the PR.

I'll add an inline comment and tag you, you've missed one thing in #7289 that we need to cover in order to match the format for both single and multisite.